### ISSUE TITLE
JAWS 2022 overly verbose with aria-roledescription on textarea

### ISSUE THREAD

**Issue by bbsmooth** [FreedomScientific/VFO-standards-support#598]
### Summary

When the aria-roledescription attribute is set on a textarea JAWS 2022 will add the attribute value to both the beginning and end of each line as you traverse the textarea with the up and down arrow keys.  This behavior is not present in JAWS 2021.

Go to https://codepen.io/bbsmooth/full/poWLgdw for a demonstration.

### Expected result

My understanding is that aria-roledescription should replace the default role of the element and thus it should be announced once, in place of the default role, when the user enters the textarea.

### Actual result

The value of the aria-roledescription attribute is being read both before and after each line in the textarea.

### Example

https://codepen.io/bbsmooth/full/poWLgdw

### Additional Information

The [monaco editor](https://github.com/Microsoft/monaco-editor) uses aria-roledescription set to "editor".

#### JAWS version and build number

2022.2112.24 
 
#### Operating System and version

Windows 10.0 (Build 10240)

#### Browser and version:

Chrome 96.0.4664.110 (Official Build) (64-bit)
Firefox 95.0.2 (64-bit)

**Comment by JAWS-test** [comment]
For me it behaves differently (tested in Chrome): JAWS 2022 does not output the aria-roledescription at all: Neither when focusing the input field with Tab, nor when navigating within the input field with arrow keys.

**Comment by bbsmooth** [comment]
Interesting. I help out on the forums at freecodecamp.org and this was first brought to my attention by a screen reader user going through the curriculum. FCC uses the monaco editor for code submission and it has aria-roledescription set to "editor". I made a quick [video to demonstrate what we are hearing](https://www.youtube.com/watch?v=ZK9wz_lb_3E).

Do you have any ideas as to what may be causing this issue for some of us?

**Comment by JAWS-test** [comment]
Unfortunately, I do not know the cause of the different output. I only know that the textarea element has many problems when outputting with JAWS: https://github.com/FreedomScientific/VFO-standards-support/issues?q=is%3Aissue+textarea

I would recommend removing the aria-roledescription. It would be better to pass the word "editor" as a label or part of the label (e.g. in the aria-label).

Regardless, the problem should be analyzed and fixed by Freedom Scientific.

**Comment by StephenBeckettTPGi** [comment]
Can confirm this behaviour (announced both the beginning and end of each line) in JAWS 2022/Chrome, Edge and Firefox. Does not occur in JAWS 2021.

**Comment by bbsmooth** [comment]
I just tested JAWS 2023.2302.15 with Chrome/FF/Edge and this issue appears to be fixed as I am not able to replicate it with my codepen link above.
